rosa_install_access_delete_clusters/rosa-sts-config-identity-providers.adoc

```diff
@@ -8,7 +8,7 @@ toc::[]
 
 After your {product-title} (ROSA) cluster is created, you must configure identity providers to determine how users log in to access the cluster.
 
-The following topics describe how to configure an identity provider using {cluster-manager} console. Alternatively, you can use the `rosa` CLI to configure an identity provider and access the cluster.
+The following topics describe how to configure an identity provider using {cluster-manager} console. Alternatively, you can use the {product-title} (ROSA) CLI (`rosa`) to configure an identity provider and access the cluster.
```

Comment:
just ROSA CLI, product title is expanded earlier

Reply:
Maybe, "ROSA CLI (`rosa`)" here?